.gitlab-ci.yml

```diff
@@ -103,7 +103,9 @@ test:
   <<:                              *docker-env
   <<:                              *common-refs
   script:
-    - cargo test
+    - cargo install cargo-nextest --locked
```

Comment:
This is not necessary, the image already contains `cargo-nextest` (not the latest version though)

Reply:
Thanks! I'll remove it, does `cargo-hack` also exist on the image?

Reply:
No, we can add it if needed. But it might take some time when it reaches the production image